### ISSUE TITLE
Android | Issue while getting paymentIntent data from stripe.

### ISSUE THREAD

**Issue by nirmalsolanki6** [stripe/stripe-terminal-react-native#400]
**Describe the bug**
I have created react native app using react-native-stripe-terminal SDK. and used the **createPaymentIntent** method of stripe SDK and passed the customer on the request payload of the createPaymentIntent method. But when I checked paymentIntent and saw paymentIntent on the stripe dashboard I am getting customer null

**To Reproduce**

1. Call createPaymentIntent
2. Pass the id of the customer in the request payload.
3. Check the paymentIntent event on the stripe dashboard.
4. See the customer

**Screenshots**
![image](https://user-images.githubusercontent.com/91239478/195799264-150e106b-a912-46bd-98fd-8200f649b5ba.png)
![image](https://user-images.githubusercontent.com/91239478/195799469-51b16d8b-d847-4a3f-aaec-a1bfb638d356.png)

**Expected behavior**
The customer will be set on paymentIntent Event.

**Smartphone (please complete the following information):**

- Device: Samsung F23 
- OS: Android 12

**Additional context**

Here is the code and request body where I have passed the customer.

```
 createdPaymentIntent = await createPaymentIntent({
  "amount": 99999,
  "applicationFeeAmount": 2690.2231,
  "currency": "usd",
  "customer": "cus_MWJfTMzassB9yc",
  "paymentMethodTypes": [
    "card_present"
  ],
  "setupFutureUsage": "off_session"
});
```

Here is the response:

```
{
  "error": "undefined",
  "paymentIntent": {
    "charges": [],
    "amount": 99999,
    "created": "1665735190000",
    "currency": "usd",
    "description": null,
    "id": "pi_3Lsj10Pajr3xPxdi0KqVNSsl",
    "status": "requiresPaymentMethod"
  }
}

```

**Comment by billfinn-stripe** [comment]
Leaving this open until we release the fix.

**Comment by mindy-stripe** [comment]
Closing as this was fixed in [v0.0.1-beta.11](https://github.com/stripe/stripe-terminal-react-native/releases/tag/v0.0.1-beta.11)
